src/utils/RealtimeFeedUtils.js

```diff
@@ -172,8 +87,33 @@ function getDelayInformation(stopID: String, tripID: String, rtf: Object): ?Obje
   };
 }
 
+function getVehicleInformation(
+  routeID: String,
+  tripID: String,
+  vehicles: Object,
+): ?Array<Object> {
+  // vehicles param ensures the vehicle tracking information doesn't update in
+  // the middle of execution
+  if (!routeID
+    || !tripID
+    || !vehicles
+    || vehicles === {}) {
```

Comment:
would `vehicles === {}` not be caught by `!vehicles`?

Reply:
for some reason this is what `getDelayInformation` is checking with the `!rtf` and `rtf === {}` so i just copied it over... but what you're saying makes more sense to me so i'll go do some investigation

Reply:
it does not catch it im pretty sure so you need the check